### PR TITLE
Add Document to Breadcrumb Component

### PR DESCRIPTION
**Pull request by rinjInTokyo**
add document to breadcrumb component

### CODE REVIEW

File: src/components/breadcrumb/daikin-breadcrumb.ts [hunk 1]
```diff
@@ -9,6 +9,24 @@ import { createRef, ref, type Ref } from "lit/directives/ref.js";
 import tailwindStyles from "../../tailwind.css?inline";
 import type { DaikinBreadcrumbItem } from "../breadcrumb-item/daikin-breadcrumb-item";
 
+/**
+ * You can use breadcrumb component and breadcrumb-item component to display the current location within a hierarchy. And allow user going back to states higher up in the hierarchy.
```
Comment (yodas7) [inline src/components/breadcrumb/daikin-breadcrumb.ts:13]: When referring to components in a document, please use the format `daikin-*` to linkify it.

```suggestion
 * `daikin-breadcrumb` is a component for representing a breadcrumb list, and is used together with the `daikin-breadcrumb-item` component.
 *
 * Hierarchy:
 * - `daikin-breadcrumb` > `daikin-breadcrumb-item`
```

**Review by yodas7 — CHANGES_REQUESTED** [review]

**Review by yodas7 — CHANGES_REQUESTED** [review]
The descriptions of some properties could be improved.

**daikin-breadcrumb**

> Specify overflow
> when visible the breadcrumb will not be omitted even breadcrumb-items total width exceed container width

```
Specifies overflow.
If `ellipsis`, some breadcrumb items will be omitted when the total width exceeds the container width.
```

> Specify whether the last of breadcrumb-item should show slash

```
Whether the last breadcrumb item should have trailing slash.
```

**daikin-breadcrumb-item**

> Specify the link should be hidden when ellipsis mode

```
Whether the link should be hidden when ellipsis mode.
Set automatically by `daikin-breadcrumb`.
```

> Specify the link should show slash at the end or not

```
Whether the link should have trailing slash.
Set automatically by `daikin-breadcrumb`.
```

> Specify link variant

```
Specifies the display content.
If `ellipsis`, the "..." will be displayed instead of the link.
Set automatically by `daikin-breadcrumb`.
```

File: src/components/breadcrumb-item/daikin-breadcrumb-item.ts [hunk 2]
```diff
@@ -83,13 +102,15 @@ export class DaikinBreadcrumbItem extends LitElement {
   target: string | null = null;
 
   /**
-   * Specify the link should show slash at the end or not
+   * Whether the link should have trailing slash.
```
Comment (yodas7) [inline src/components/breadcrumb-item/daikin-breadcrumb-item.ts:105]: Sorry the slash is not in the link.

```suggestion
   * Whether the slash after the link should shown.
```

**Review by yodas7 — CHANGES_REQUESTED** [review]

**Review by yodas7 — APPROVED** [review]
LGTM.

**Review by ukkary — APPROVED** [review]

**Pull request merged**